includes/admin/class-sensei-setup-wizard-pages.php

```diff
@@ -155,33 +141,62 @@ private function get_learner_courses_page_template() {
 
 <!-- wp:query-pagination-next {"fontSize":"small"} /-->
 <!-- /wp:query-pagination --></div>',
-						],
-						'attrs'        => [
-							'queryId'       => 0,
-							'query'         => [
-								'postType' => 'course',
-								'perPage'  => 10,
-								'offset'   => 0,
-								'inherit'  => false,
-								'sticky'   => '',
-								'pages'    => 0,
-								'order'    => 'desc',
-								'orderBy'  => 'date',
-								'author'   => '',
-								'search'   => '',
-								'exclude'  => [],
-							],
-							'displayLayout' => [
-								'type' => 'list',
-							],
-							'className'     => 'wp-block-sensei-lms-course-list wp-block-sensei-lms-course-list--is-list-view',
-						],
+				],
+				'attrs'        => [
+					'queryId'       => 0,
+					'query'         => [
+						'postType' => 'course',
+						'perPage'  => 10,
+						'offset'   => 0,
+						'inherit'  => false,
+						'sticky'   => '',
+						'pages'    => 0,
+						'order'    => 'desc',
+						'orderBy'  => 'date',
+						'author'   => '',
+						'search'   => '',
+						'exclude'  => [],
 					],
-				]
+					'displayLayout' => [
+						'type' => 'list',
+					],
+					'className'     => 'wp-block-sensei-lms-course-list wp-block-sensei-lms-course-list--is-list-view',
+				],
+			],
+		];
+
+		/**
+		 * Filter the learner courses page template when auto-creating it
+		 * through setup wizard.
+		 *
+		 * @hook  sensei_learner_courses_page_template
+		 * @since 3.13.1
+		 *
+		 * @deprecated $$next_version$$
+		 *
+		 * @param {array} $blocks Blocks array.
+		 *
+		 * @return {array} Blocks array.
+		 */
+		$blocks = apply_filters( 'sensei_learner_courses_page_template', $blocks );
```

Comment:
Oh, whoops! I missed that this was an existing hook! To deprecate a hook, we need to use `apply_filters_deprecated`, but I think it would be easier to just revert this commit. 🙂